.github/actions/test-khiops-install/action.yml

```diff
@@ -9,14 +9,14 @@ runs:
     - name: Print status for debugging
       shell: bash
       # We force the process number to 4 because the runner has only 2 proc.
-      # With 2 proc khiops-env switches to serial and mpi is not used.
+      # With 2 proc khiops_env switches to serial and mpi is not used.
```

Comment:
Why did we need to rename the script?

Reply:
It is simpler this way and avoid mistakes when calling it: khiops_env on all platforms

Reply:
Ok.